internal/commands/document.go

```diff
@@ -214,6 +230,18 @@ func getDocumentation(path string, outputDirectory string) (map[rego.Severity][]
 	return documents, nil
 }
 
+func labelSelectorDocString(selector *metav1.LabelSelector) string {
+	var result string
+	for k, v := range selector.MatchLabels {
+		result += fmt.Sprintf("%s=%s, ", k, v)
+	}
+	for _, expr := range selector.MatchExpressions {
+		result += fmt.Sprintf("%s %s %v, ", expr.Key, expr.Operator, expr.Values)
+	}
+
+	return strings.TrimSuffix(strings.TrimSpace(result), ",")
```

Comment:
why not simply `strings.TrimSuffix(result, ", ")`?

Reply:
You're correct, that would be more succinct. I was on auto-pilot I guess :) Fixed in the latest that I'll be pushing momentarily.